projects/dotcms-field-elements/src/utils/errorHandling.tsx

```diff
@@ -0,0 +1,62 @@
+const fieldValidationMap = {
```

Comment:
Should this file be named after "prop" like `propValidationHandling` or something like that?